version.sbt

```
@@ -1 +1 @@
-ThisBuild / version := "1.2.0"
+ThisBuild / version := "2.0.0"
```

```suggestion
ThisBuild / version := "2.0.0-SNAPSHOT"
```
to set sbt-release ready for releasing 2.0.0